app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ApplicationsController < ApplicationController
+  def index
+    @applications = Application.all
+  end
+
+  def show
+    @application = Application.find(params[:id])
+    @pets = Pet.search(params[:add_pet_name]) 
```

Comment:
I just want to clarify one thing for the `search` method that you are calling on Pet since I didn't hear it mentioned in the video. This method does not come pre-packaged with AR and is actually a user-defined method that is defined in the ApplicationRecord model:

```ruby
class ApplicationRecord < ActiveRecord::Base
  primary_abstract_class
  
  def self.search(search_params)
    where("name ILIKE ?", "%#{search_params}%")
  end
end
```

Because all models inherit from ApplicationRecord, we can also then use it on Pet.